document/CoreFormat.mdk

```diff
@@ -1743,13 +1729,138 @@ A `codeFlow` object may contain a property named `tags`, whose value is a tag se
 A `codeFlow` object may contain a property named `properties`, whose value is a property bag.
 This allows tools to include information about the code flow that is not explicitly specified in the SARIF format.
 
+## `stack` object { #stack }
+
+A `stack` object describes a single call stack.
+A call stack is a sequence of nested function calls, each of which is referred to as a stack frame.
+
+### `message` property { #stack-message }
+
+A `stack` object may contain a property named `message` whose value is a string
+containing a message relevant to this call stack.
+
+### `frames` property { #stack-frames }
+
+A `stack` object shall contain a property named `frames` whose value is an array of one or more `stackFrame` objects (&sect;[#stackFrame]).
+This array shall include every function call in the stack for which the tool has information,
+and the entries that are present shall occur in order with the innermost call first and the outermost call last.
```

Comment:
disagree. 'most recent' is entirely clear. 'innermost' requires some mutually agreed on mental model that might not coincide with the readers.